apps/meteor/client/components/Omnichannel/modals/CloseChatModal.tsx

```diff
@@ -77,7 +78,7 @@ const CloseChatModal = ({
 		({ comment, tags, transcriptPDF, transcriptEmail, subject }): void => {
 			const preferences = {
 				omnichannelTranscriptPDF: !!transcriptPDF,
-				omnichannelTranscriptEmail: !!transcriptEmail,
+				omnichannelTranscriptEmail: alwaysSendTranscript ? true : !!transcriptEmail,
```

Comment:
Actually, dont know. Wanted to be more declarative here to put that when the setting is true, then it will always be true.

Open to change but feel kinda a pref thing